src/commands/protocols/update/receiver/v1.0.0/v1-0-0-handle-update-init-command.js

```diff
@@ -8,7 +8,6 @@ import {
 class HandleUpdateInitCommand extends HandleProtocolMessageCommand {
     constructor(ctx) {
         super(ctx);
-        this.publishService = ctx.updateService;
```

Comment:
are we sure that this is not needed ? Have you verified that all functions in parent classes are not using operationService / publishService ?

Reply:
Yes, we are not using it. But even if we are it should be handled in constructor of parent classes, as we are always calling `super(ctx)`.

Reply:
Parent classes are shared with other routes, such as get and publish, so the operation service needs to be initialized in child classes. For other commands we have `this.operationService = ctx.UpdateService` for example